examples/meta/src/evaluation/accuracy_multiclass.sg.in

```diff
@@ -0,0 +1,10 @@
+File f_labels_train = csv_file("@SHOGUN_DATA@/classifier_binary_2d_linear_labels_train.dat")
```

Comment:
the kernel data changes shouldnt be there. you need to rebase or set the data submodule to HEAD and then apply your changes again

Reply:
aha so just set the data submodule to head and copy the two files there?